notebooks/index.ipynb

```diff
@@ -38,7 +38,7 @@
    "metadata": {},
```

Comment:
<p>Why not 2?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlucool/jupyterlab-execute-time/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Because....